plumgrid_playbooks/enable_pg_extensions.yml

```diff
@@ -19,16 +19,9 @@
   tasks:
     - name: Remove previous PLUMgrid sources.list
       file:
-        path: /etc/apt/sources.list.d/plumgrid.list
+        path: "/etc/apt/sources.list.d/plumgrid.list"
         state: absent
 
-    - name: Add PLUMgrid openstack to repo
-      lineinfile:
-        dest: /etc/apt/sources.list.d/plumgrid.list
-        line: "deb {{ plumgrid_repo }}/openstack/deb/{{ openstack_version }} openstack-{{ openstack_version }} {{ plumgrid_component }}"
-        state: present
-        create: yes
-
     - name: Copy PLUMgrid GPG-key file
```

Comment:
Hi Arbab, thanks for your comments.
Every release of our software has a different GPG key. Making it static would mean we have to manually change it every time for a new release. Also, once we make it static, we wont be able to install previous version of our software with these playbooks.